mne/viz/evoked.py

```diff
@@ -1926,9 +1926,12 @@ def plot_compare_evokeds(evokeds, picks=None, colors=None,
     ----------
     evokeds : instance of mne.Evoked | list | dict
         If a single Evoked instance, it is plotted as a time series.
+        If a list of Evokeds, the contents are plotted with their
+        ``.comment`` attributes used as condition labels. If no comment is set,
+        the index of the respectiv Evoked the list will be used instead,
+        starting with `1` for the first Evoked.
```

Comment:
/home/circleci/project/mne/viz/evoked.py:docstring of mne.viz.plot_compare_evokeds:8: WARNING: py:obj reference target not found: 1
```suggestion
        starting with ``1`` for the first Evoked.
```